Description: Some fixes within documentation, Advice about passwords-on-commandline
Origin: old upstream version
Author: Philipp Matthias Hahn <pmhahn@debian.org>
DEP: 3
--- a/mytop
+++ b/mytop
@@ -1907,30 +1907,33 @@
 
 =over
 
-=item B<-u> or B<-user> username
+=item B<-u> or B<--user> I<username>
 
-Username to use when logging in to the MySQL server. Default: ``root''.
+Username to use when logging in to the MySQL server. Default: ``B<root>''.
 
-=item B<-p> or B<-pass> or B<-password> password
+=item B<-p> or B<--pass> or B<--password> I<password>
 
 Password to use when logging in to the MySQL server. Default: none.
 
-=item B<-h> or B<--host> hostname[:port]
+WARNING: This is insecure as the password is visible for anyone.
+See B<--prompt> instead!
+
+=item B<-h> or B<--host> I<hostname>[B<:>I<port>]
 
 Hostname of the MySQL server. The hostname may be followed by an
 option port number. Note that the port is specified separate from the
-host when using a config file. Default: ``localhost''.
+host when using a config file. Default: ``B<localhost>''.
 
-=item B<--port> or B<-P> port
+=item B<--port> or B<-P> I<port>
 
 If you're running MySQL on a non-standard port, use this to specify
-the port number. Default: 3306.
+the port number. Default: B<3306>.
 
-=item B<-s> or B<--delay> seconds
+=item B<-s> or B<--delay> I<seconds>
 
-How long between display refreshes. Default: 5
+How long between display refreshes. Default: B<5>
 
-=item B<-d> or B<--db> or B<--database> database
+=item B<-d> or B<--db> or B<--database> I<database>
 
 Use if you'd like B<mytop> to connect to a specific database by
 default. Default: ``test''.
@@ -1939,14 +1942,14 @@
 
 In batch mode, mytop runs only once, does not clear the screen, and
 places no limit on the number of lines it will print. This is suitable
-for running periodically (perhaps from cron) to capture the
+for running periodically (perhaps from B<cron>) to capture the
 information into a file for later viewing. You might use batch mode in
 a CGI script to occasionally display your MySQL server status on the
 web.
 
 Default: unset.
 
-=item B<-S> or B<--socket> /path/to/socket
+=item B<-S> or B<--socket> I</path/to/socket>
 
 If you're running B<mytop> on the same host as MySQL, you may wish to
 have it use the MySQL socket directly rather than a standard TCP/IP
@@ -1961,7 +1964,7 @@
 
 =item B<--header> or B<--noheader>
 
-Sepcify if you want the header to display or not. You can toggle this
+Specify if you want the header to display or not. You can toggle this
 with the B<h> key while B<mytop> is running.
 
 Default: header.
@@ -1987,7 +1990,7 @@
 Specify if you want to be prompted to type in your database password.
 This provides a little bit more security since it not only prevents
 the password from viewable in a process list, but also doesn't require
-the password to be stored in plain text in your ~/.mytop config file.
+the password to be stored in plain text in your C<~/.mytop> config file.
 You will B<only> be prompted if a password has not been specified in
 your config file or through another command line option.
 
@@ -2145,8 +2148,8 @@
 
 The B<s> key has a command-line counterpart: B<-s>.
 
-The B<h> key has two command-line counterparts: B<-header> and
-B<-noheader>.
+The B<h> key has two command-line counterparts: B<--header> and
+B<--noheader>.
 
 =head1 BUGS
 
@@ -2300,12 +2303,12 @@
 
 =item jon r. luini <falcon@chime.com>
 
-Supplied a patch that formed the basis for C<-prompt> support.  Sean
+Supplied a patch that formed the basis for C<--prompt> support.  Sean
 Leach <sleach@wiggum.com> submitted a similar patch.
 
 =item Yogish Baliga <baliga@yahoo-inc.com>
 
-Supplied a patch that formed the basis for C<-resolve> support.
+Supplied a patch that formed the basis for C<--resolve> support.
 
 =item Per Andreas Buer <perbu@linpro.no>
 
